src/Ui.GraphiQL/GraphiQLOptions.cs

```diff
@@ -35,5 +35,16 @@ public class GraphiQLOptions
         /// Gets or sets a delegate that is called after all transformations of the GraphiQL UI page.
         /// </summary>
         public Func<GraphiQLOptions, string, string> PostConfigure { get; set; } = (options, result) => result;
+
+        /// <summary>
+        /// Enables the header editor when <c>true</c>.
+        /// Not supported when ExplorerExtensionEnabled is <c>true</c>.
+        /// </summary>
+        public bool HeaderEditorEnabled { get; set; } = true;
```

Comment:
```suggestion
        public bool HeadersEditorEnabled { get; set; } = true;
```
?

Reply:
I would chose the original naming from graphiql over the plural: https://github.com/graphql/graphiql/blob/08250feb6ee8335c3b1ca83a912911ae92a75722/packages/graphiql/src/components/GraphiQL.tsx#L186

Reply:
OK. Please copy link to this "source-settings" into xml comments for GraphiQLOptions.